ares/n64/cpu/cpu.cpp

```diff
@@ -147,8 +147,6 @@ auto CPU::instructionEpilogue() -> s32 {
     icache.step(ipu.pc, devirtualizeFast(ipu.pc));
   }
 
-  ipu.r[0].u64 = 0;
```

Comment:
Actually: what about when the interpreter enabled?

Reply:
A good catch. I wrapped it in a constexpr check like the icache test above it.